src/signals/incident/services/custom-validators/index.js

```diff
@@ -44,3 +50,8 @@ export const validatePhoneNumber = control => {
 
   return { custom: 'Ongeldig telefoonnummer, alleen cijfers, spaties, haakjes, + en - zijn toegestaan.' };
 };
+
+export const validateRequired = message =>
+  function customRequired({ value }) {
```

Comment:
if the name here would be `required` it will fit the required validation pattern